packages/macro/test/fixtures/jsx-keep-forced-newlines.expected.js

```diff
@@ -0,0 +1,2 @@
+import { Trans as _Trans } from "@lingui/react"
+;<_Trans id={"<stripped>"} message={"Keep multiple\nforced\nnewlines!"} />
```

Comment:
i extracted this test cases in the test file cases because it was quite hard to understand is escaping working correctly or not when the code is inside js string